README.md

```diff
@@ -126,34 +147,95 @@ The idea is that a single large commit keeps the master branch neater than lots
     Commit the staged snapshot regularly. Instead of launching a text editor, you can use `<message>` as a description of this commit. 
 3. Checkout and pull from master branch:
 
+        # Move into local master branch
```

Comment:
Let's add a little more color to why step 3 is needed:
To ensure your working branch is up to date with the changes pushed to master by other team members so if there are conflicting changes, you can resolve them and re-test your code

Reply:
You'd also need step 3.5 to checkout your working branch before merging master into it as specified in step 4.